### PR TITLE
Hyperparameter tuning example notebook

### LINKED ISSUE #718: Hyperparameter tuning with optuna notebook
Make a common workflows notebook that shows tuning with optuna. 

Three different starting learning rates 
0.1
0.05
0.01

Three different LR schedulers. 

Default LR scheduler w/ gamma=1
Default LR scheduler w/ gamma=0.9
CosineAnnealing

### PR DESCRIPTION
**Pull request by drewoldag**
Going to write the same example for a handful of different tuning libraries.
- [x] Optuna
- [ ] ~RayTune~ Striking this one, Hyrax doesn't immediately work with it. 
- [x] Hyperopt
- [ ] ~weights and biases~ Striking this one, it requires API keys and other external set up.

Added notebook to common workflows page.
Added `../docs/_static/screenshots` directory for general use.

## Change Description
Closes #718

### CODE REVIEW

**Review by mtauraso — APPROVED** [review]
LGTM
